backend/routes/indexer.go

```diff
@@ -283,6 +271,9 @@ func processNFTMintedEvent(event IndexerEvent, w http.ResponseWriter) {
 	// TODO: Ws message to all clients
 
 	WriteResultJson(w, "NFT mint indexed successfully")
+
+	sendNFTWebSocketMessage(w, tokenId)
```

Comment:
In order to notify the user on who minted the NFT, add the minter to the websocket message.